.gitignore

```diff
@@ -30,6 +30,7 @@ desktop.ini
 # Editors / IDEs
 .floo
 .flooignore
+.brackets.json
```

Comment:
Unrelated, but it's helpful for me. Just close your eyes and pretend it's not part of this commit.